app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+          annotationType: "CATEGORICAL",
+          name: data.name,
+          values: data.values,
+          id: initialAnnotationConfig?.id || "",
+          optimizationDirection: data.optimizationDirection,
+          description: data.description,
+        };
+        onAddAnnotationConfig(config, { onCompleted, onError });
+        break;
+      }
+      case "CONTINUOUS": {
+        const config: AnnotationConfigContinuous = {
+          annotationType: "CONTINUOUS",
+          name: data.name,
+          lowerBound: data.lowerBound,
+          upperBound: data.upperBound,
+          id: initialAnnotationConfig?.id || "",
+          optimizationDirection: data.optimizationDirection,
+          description: data.description,
+        };
+        onAddAnnotationConfig(config, { onCompleted, onError });
+        break;
+      }
+      case "FREEFORM": {
+        const config: AnnotationConfigFreeform = {
+          annotationType: "FREEFORM",
+          name: data.name,
+          id: initialAnnotationConfig?.id || "",
+          description: data.description,
+        };
+        onAddAnnotationConfig(config, { onCompleted, onError });
+        break;
+      }
+    }
+  };
+  const annotationType = watch("annotationType");
+  return (
+    <Dialog
+      css={css`
+        border: none;
+        min-width: 700px;
+      `}
+    >
+      {({ close }) => (
+        <Card
+          title={
+            mode === "new" ? "New Annotation Config" : "Edit Annotation Config"
+          }
+          variant="compact"
+          bodyStyle={{ padding: 0 }}
+          extra={
+            <Button
+              onPress={close}
+              size="S"
+              css={css`
+                & {
+                  padding-left: var(--ac-global-dimension-size-50) !important;
+                  padding-right: var(--ac-global-dimension-size-50) !important;
+                }
+              `}
+            >
+              <Icon svg={<Icons.CloseOutline />} />
+            </Button>
+          }
+        >
+          <Form
+            onSubmit={(e) => {
+              handleSubmit((data) => {
+                onSubmit(data, close);
+              })(e);
+            }}
+          >
+            <View
+              minWidth="200px"
+              padding="size-200"
+              maxHeight="600px"
+              overflow="auto"
+            >
+              <Flex
+                direction="column"
+                gap="size-200"
+                className="new-annotation-dialog-container"
+              >
+                <Controller
+                  name="name"
+                  control={control}
+                  rules={{
+                    required: "Name is required",
+                  }}
+                  render={({ field, fieldState: { error } }) => (
+                    <TextField {...field} isInvalid={!!error}>
+                      <Label>Annotation Name</Label>
+                      <Input placeholder="correctness" />
```

Comment:
With placeholders, it's best practice to not put a valid value there since it looks valid. Always use some sort of text like "e.x. correctness"